app/views/review_details_non_uk_agent.scala.html

```diff
@@ -16,43 +16,49 @@
 
 @import config._
 @import helpers.pageHeadersAndError
-@this(bcMain: bcMain)
+@import views.html.helper.form
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@this(bcMain: bcMain, formHelper: FormWithCSRF, govukTable : GovukTable)
 
 @(service: String, businessDetails: models.ReviewDetails, backLink: Option[String])(implicit request: Request[AnyContent], messages: Messages, appConfig: ApplicationConfig)
 
-@import uk.gov.hmrc.play.views.html.helpers.form
 
 @bcMain(title = messages("business-review.agent-editable.title", service), service = service) {
 
   @pageHeadersAndError(backLink, "bc.business-registration-agent.text", messages("bc.business-registration.text.agent", service.toUpperCase), "bc.business-registration-agent.header", messages("business-review.agent-editable.title"))
 
-  <div class="form-group">
-    <div class="confirmation-notice">
-      <div class="icon-important"></div>
-      <div class="ated-icon-important-text" id="check-agency-details"><strong>@messages("business-review.check-agency.text")</strong></div>
+  <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <div id="check-agency-details">
+      <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+            @messages("business-review.check-agency.text")
+      </strong>
+      </div>
   </div>
 
-  <table id="business-details">
-      <tr>
-        <th scope="row" id="business-name-title">
+
+      <table id="business-details" class="govuk-table">
```

Comment:
Since it was a table alreay I thought I should keep as a table and also on the summary list description it mentions "Do not use it for tabular data or a simple list of information or tasks"